### PR TITLE
fix(maxmsp): text_meassure signature is wrong

### PR DESCRIPTION
**Pull request by FrancescElies**
[Here](https://docs.cycling74.com/max8/vignettes/jsmgraphics) the docs to that function

### CODE REVIEW

File: types/maxmsp/maxmsp-tests.ts [hunk 1]
```diff
@@ -358,7 +358,8 @@ myMGraphics.show_text('Hello');
 myMGraphics.text_path('Hello');
 const fontExtents = myMGraphics.font_extents();
 post(fontExtents);
-const textMeasure = myMGraphics.text_measure();
+const textMeasure = myMGraphics.text_measure("my-string");
+const [widht, height] = textMeasure;
```
Comment (twhiston) [inline types/maxmsp/maxmsp-tests.ts:362]:
```suggestion
const [width, height] = textMeasure;
```

**Review by twhiston — CHANGES_REQUESTED** [review]
Just one pedantic change around the spelling of width, otherwise this is good to go.

**Review by twhiston — APPROVED** [review]
Thanks very much 😁

**Comment by FrancescElies** [comment]
Ready to merge

**Pull request merged**